odufrn_api_py/modules/Env.py

```diff
@@ -0,0 +1,84 @@
+import json
+import requests
+from abc import ABC
+
+
+class Env(ABC):
+    """Classe com os atributos e métodos responsáveis pelo
+    funcionamento do pacote.
+
+    Atributos
```

Comment:
A classe possui vários atributos, não apenas esses. Esses atributos são passados no `__init__`, portanto faz sentido essa parte da documentação está no `__init__`. Creio que se você for colocar todos os atributos da classe aqui, ficará um textão.